### DISCUSSION TITLE
Czy ten kod jest potrzebny

### DISCUSSION THREAD

**Discussion by itchystone** [emkarcinos/WMIAdventure#409] [Codding]
Można to scalić, czy zostawić? (Plik CardsCreator.js)

![image](https://user-images.githubusercontent.com/43492764/126635595-3fc7c440-1028-4887-b8bb-78cea1a235e6.png)

**Comment by itchystone** [comment]
Tak samo tutaj zauważyłem, że ten kod różni się tylko jedną małą instrukcją:
![image](https://user-images.githubusercontent.com/43492764/126647726-2ca1a9c7-a0e8-47f6-aacb-35ad713fa56a.png)

Mogę to scalić w jedno za pomocą 
![image](https://user-images.githubusercontent.com/43492764/126647890-08ced6eb-1c37-47df-b02f-8524e12c8e13.png)
czy w przyszłości potrzebny jest ten podział na `if else`?

**Reply by mattyl006** [reply]
Można scalić, sam to chciałem zrobić, ale się wstrzymywałem, bo wtedy była ta sytuacja co wszystko nam się sypało, więc się cykałem. Wcześniej to miało sens, ale jak doszła ta baza danych z kartami czekającymi na akceptacje, to tutaj też mamy post, a nie put, więc request ten sam przy tworzeniu jak i edycji.

**Comment by mattyl006** [comment]
To isMobile @dkorybalski dodał, też wydaje mi się niepotrzebne.

**Comment by mattyl006** [comment]
Można taska zrobić na ten refactor. Zrobić ten refactor i sprawdzić czy nic się nie zepsuło.

**Comment by mattyl006** [comment]
Teraz jak robię te levele na desktopa to wywaliłem te isMobile, bo chyba nie działało nawet

**Reply by itchystone** [reply]
Spoko
